gno.land/cmd/gnoland/main.go

```diff
@@ -314,3 +309,103 @@ func loadGenesisBalances(path string) []string {
 	}
 	return balances
 }
+
+// listGnoPkgs lists all gno packages in the given root directory.
+func listGnoPkgs(root string) ([]pkg, error) {
```

Comment:
Helper needed in gnovm/pkg/gnomod for #743.

Reply:
Resolved-by: 10330b5030e813008fcd8617de36158a7d5a9896